src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -26,7 +25,7 @@ object BlockingProcess {
    * @param stdout a `org.reactivestreams.Publisher` for the standard output stream of the process
    * @param stderr a `org.reactivestreams.Publisher` for the standard error stream of the process
    */
-  case class Started(stdin: Subscriber[ByteString], stdout: Publisher[ByteString], stderr: Publisher[ByteString])
+  case class Started(stdin: Sink[ByteString, Future[Long]], stdout: Source[ByteString, Future[Long]], stderr: Source[ByteString, Future[Long]])
```

Comment:
I'm simply following the returns types provided by Akka Stream itself - `Source.inputStream()` returns `Source[ByteString, Future[Long]]` while `Sink.outputStream()` returns `Sink[ByteString, Future[Long]]`.